### DISCUSSION TITLE
Problem after upgrade of netbox

### DISCUSSION THREAD

**Discussion by dionrowney** [mattieserver/netbox-topology-views#584] [General]
I just upgraded netbox to 4.1.6 and along with that the topology views plugin.

All seemed fine in my dev and test setups but in prod it doesnt display anything and I get this error in javasript console:

app.js:23  Uncaught TypeError: Cannot read properties of undefined (reading '0')
    at app.js:23:229670
    at app.js:23:234653
    at app.js:23:236102
(anonymous) @ app.js:23
(anonymous) @ app.js:23
(anonymous) @ app.js:23

All browsers exhibit same behavior

Images displays fine.  permissions onn static folder seemms fine too

any thoughts?

**Comment by dreng** [comment]
I skipped 4.1.6 in my production environment and updated from 4.1.5 to 4.1.7, which seem to run without issues. Would you please make sure that the problem persists in 4.1.7?

I tend to guess that there's an issue with some data in your production database.

**Comment by dionrowney** [comment]
Seems I had to just restart the gunicorn process again.  Seems fine now.  my bad.
